apps/mobile/src/slices/auth/auth.slice.ts

```diff
@@ -2,15 +2,21 @@ import { createSlice } from "@reduxjs/toolkit";
 
 import { DataStatus } from "~/libs/enums/enums";
 import { type ValueOf } from "~/libs/types/types";
+import {
+	type UserSignInResponseDto,
+	type UserSignUpResponseDto,
+} from "~/packages/users/libs/types/types";
 
-import { signUp } from "./actions";
+import { signIn, signUp } from "./actions";
 
 type State = {
 	dataStatus: ValueOf<typeof DataStatus>;
+	user: null | UserSignInResponseDto | UserSignUpResponseDto;
```

Comment:
```suggestion
	user: null | UserDto;
```
ResponseDto contains token, we don't need to save it in redux